### ISSUE TITLE
get error when modify a rust file Error executing bad argument #1 to 'pairs' (table expected, got nil)

### ISSUE THREAD

**Issue by fcying** [hrsh7th/cmp-buffer#27]
```
Error executing vim.schedule lua callback: ...ged/pack/packer/opt/cmp-buffer/lua/cmp_buffer/buffer.lua:130: Error executing lua: ...ged/pack/packer/opt/cmp-buffer/lua/cmp_buffer/buffer.lua:254: bad argument #1 to 'pairs' (table expected, got nil)
```

```
wget https://github.com/epilys/rsqlite3/raw/main/src/lib.rs
nvim lib.rs
view
modify 
error jump.
```

![image](https://user-images.githubusercontent.com/106934/142582190-6b51b71e-ac30-48f5-b08c-bf56e933ff5a.png)

nvim version: 0.5

**Comment by dmitmel** [comment]
This will be fixed by #23, but I can make a PR with a patch for just this bug.

**Comment by fcying** [comment]
Thanks, look forward to the completion of the PR.
